File: lesson5/python3/homework/homework5_6.py
#!/usr/bin/python3
#: lesson5/python3/homework/homework5_6.py


def number_hex(numb_hex):
    numb_h = hex(numb_hex)
    numb_i = numb_h.replace('x', '0')
    print(numb_i)


numb_n = int(input("Введите число -> "))
number_hex(numb_n)

File: lesson5/c_c++/homework/homework5_6.cpp
//: lesson5/c_c++/homework/homework5_6.cpp
#include <iostream>

void numbHex(int numb_hex) {
    std::cout << std::hex << numb_hex << std::endl;
    // Добавляем std::uppercase, если хотим в верхнем регистре.
}

int main() {
    int numb_n;
    std::cout << "Введите число -> ";
    std::cin >> numb_n;
    numbHex(numb_n);
    return 0;
}
